fakeresponse.js

```diff
@@ -88,12 +92,34 @@ var FakeResponse = {
                 if(item.queryParams && !FakeResponse.matchRegex(item.queryParams, uri.query)) return false;
                 if(item.payload && !FakeResponse.matchRegex(item.payload, payload)) return false;
                 if(item.requiredHeaders && !FakeResponse.matchRegex(item.requiredHeaders, headers)) return false;
+                if(item.verb && !(item.verb==verb)) return false;
+                //Revisar para que sea dividido por el modulo en lugar de solo la primera vez que coincidan
                 if (item.at) return (item.numCalls === item.at);
                 return true;
             }
             return false;
         }).sort(FakeResponse.compareMatches)[0] || null;
     },
+    
+    /* Filters all items that match the URL and then tries to check if there is a specific behavior for the Nth call on the same endpoint */
+    matchDel: function (uri, responseCode, verb) {
```

Comment:
this method is too similar to "match". Either we should merge them or make one of them use the other.